modules/installing-rhv-setting-up-ca-certificate.adoc

```diff
@@ -6,7 +6,7 @@
 [id="installing-rhv-setting-up-ca-certificate_{context}"]
 = Setting up the CA certificate for {rh-virtualization}
 
-Download the CA certificate from the {rh-virtualization-first} Manager and set it up on the installation machine.
+Unless you are using the {rh-virtualization-first} Manager to install {product-title}, download the CA certificate from the {rh-virtualization-first} Manager and set it up on the installation machine.
```

Comment:
I found this a little confusing, if the user is using RHV to install OCP, it needs to download the CA certificate used by RHV to able to perform the installation